Source/MSBuild.Community.Tasks/MSBuild.Community.Tasks.Targets

```diff
@@ -3,7 +3,7 @@
   <!-- $Id$ -->
 
   <PropertyGroup>
-    <MSBuildCommunityTasksPath Condition="'$(MSBuildCommunityTasksPath)' == ''">$(MSBuildExtensionsPath)\MSBuildCommunityTasks</MSBuildCommunityTasksPath>
+    <MSBuildCommunityTasksPath Condition="'$(MSBuildCommunityTasksPath)' == ''">$(MSBuildThisFileDirectory)</MSBuildCommunityTasksPath>
```

Comment:
Since the .dll sits next to the .targets file, we can have the .targets file locate the .dll by just looking at its siblings.  This solves the problem of NuGet packages being in a path that changes with every version.